src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Alert Rule Action
+sidebar_order: 3
+description: "Learn about the alert rule action UI component."
```

Comment:
> I think this can be a bit spicier 🌶️
> 
> > Learn how to set up Alert Rule Actions in the Sentry UI to get custom information directly from your users with any of their triggered alerts.
> 
> or something like that. cc @imatwawana

That sentence is pretty long and I'm not sure I understand it. When you say "any of their triggered alerts" do you mean that the our customer's user has taken an action that triggers an alert set up by our customer? Or something else?

Reply:
@imatwawana I was actually curious if you knew where this even shows up, I don't see it on the page anywhere so I didn't bother spending time on them.

Reply:
> @imatwawana I was actually curious if you knew where this even shows up, I don't see it on the page anywhere so I didn't bother spending time on them.

It's only in the metadata of the page, so search engines will pick it up, but it's not visible when you're just reading the docs.

Reply:
@imatwawana to follow up and put words in @leeandher 's mouth: 

"any of their triggered alerts" means the user's alerts that have the developer's service selected as an alert rule action which have fired.

Reply:
Oh also, since it's just metadata how much effort would you suggest we put into these generally? Treat it as if it were on the page or just slam something in there with key words?

Reply:
> @imatwawana to follow up and put words in @leeandher 's mouth:
> 
> "any of their triggered alerts" means the user's alerts that have the developer's service selected as an alert rule action which have fired.

That is an incredible mouthful, so I'd say let's leave that part out. And to answer your other question, I wouldn't spend a ton of time on them. They should make sense to humans, but as long as they're fundamentally accurate, we're good.